client/index.jade

```diff
@@ -5,15 +5,15 @@ html(ng-app="bitcampApp")
     meta(http-equiv="X-UA-Compatible", content="IE=10,chrome=1")
     meta(charset="utf-8")
     base(href="/")
-    title Bitcamp 2015
-    meta(name="description", content="36-hour hackathon at the University of Maryland. Spring 2015.")
+    title Bitcamp 2014
+    meta(name="description", content="36-hour hackathon at the University of Maryland. Spring 2014.")
     meta(name="viewport", content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no")
 
-    meta(property="og:site_name",   content="Bitcamp 2015")
-    meta(property="og:title",       content="Bitcamp 2015")
+    meta(property="og:site_name",   content="Bitcamp 2014")
+    meta(property="og:title",       content="Bitcamp 2014")
```

Comment:
changed the title to match the url.. 2015 did make sense at the time, but it doesn't make sense for the archived 2014 post-event website